molecule_vagrant/playbooks/create.yml

```diff
@@ -25,7 +25,7 @@
         provider_override_args: "{{ item.provider_override_args | default(omit) }}"
 
         provision: "{{ item.provision | default(omit) }}"
-
+        parallel: "{{ molecule_yml.driver.provider.parallel | default(yes) }}"
```

Comment:
nitpick : maybe ``default(omit)`` ?

Reply:
Fixed in 79db427. Did the same for `provider.name`.